gen/build_deploy/bash/dcos-launch.spec

```diff
@@ -1,4 +1,12 @@
-a = Analysis(['test_util/launch.py'])
+# Warning:
+#   - hidden import must be used as there is a bug in pyinstaller
+#     https://github.com/pyinstaller/pyinstaller/issues/2185
+#   - data must be decalared explicitly if not a .py file
+#   - Building will suck up the local SSL .so and package it
+#     with the final exe. Ensure build system has OpenSSL 1.0.2g or greater
+a = Analysis(['test_util/launch.py'],
+             hiddenimports=['html.parser'],
+             datas=[('gen/ip-detect/*.sh', 'gen/ip-detect/')])
```

Comment:
ip-detect scripts can be left off for now, but they will be required when we do VPC installs (which this PR originally included and hence this line). So, I would just rather leave this here as onprem will be one of the first expansions for this

gen is not yet invoked directly by dcos-launch so this missing yaml is not a problem currently